packages/nodes-base/nodes/Airtop/actions/node.type.ts

```diff
@@ -2,8 +2,9 @@ import type { AllEntities } from 'n8n-workflow';
 
 type NodeMap = {
 	session: 'create' | 'terminate';
-	window: 'create' | 'close' | 'getScreenshot' | 'query';
+	window: 'create' | 'close' | 'getScreenshot' | 'load' | 'query';
 	extraction: 'getPaginated';
```

Comment:
missing 'scrape' here under `extraction` and we should also move `query` to under extraction.